From 086bc0f59757831f74ac7cbd9daed42c8d655a3c Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Fri, 11 Sep 2020 10:32:28 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1109-38-v10-spi-nand: disable nandc

SPI NAND is driven by SFC, so disable nandc because do not need

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I47027c0118dadbede3b47d0d25d49ff47bde306a
---
 arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
index 36824727f0f7..710e3a78189d 100644
--- a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
@@ -186,10 +186,6 @@
 	};
 };
 
-&nandc {
-	status = "okay";
-};
-
 &npu {
 	npu-supply = <&vdd_logic_npu_vepu_fixed>;
 };
-- 
2.35.3

